nex/node_linux_amd64.go

```diff
@@ -31,3 +75,27 @@ func RunNodePreflight(ctx context.Context, logger *slog.Logger) error {
 	ctx, cancel := context.WithCancel(ctx)
 	return nexnode.CmdPreflight(Opts, NodeOpts, ctx, cancel, logger)
 }
+
+func createMetricsReader(log *slog.Logger) (metric.Reader, error) {
+	switch NodeOpts.OtelMetricsExporter {
+	case "prometheus":
+		go func() {
+			log.Info("serving metrics at localhost:8085/metrics")
+			http.Handle("/metrics", promhttp.Handler())
+			err := http.ListenAndServe(":8085", nil)
```

Comment:
We probably make this port configurable.

Reply:
Yes.  I'll add that to this PR